static/js/favicon.js

```diff
@@ -23,7 +27,11 @@ function set_favicon() {
     $("#favicon").attr("href", favicon_state.url);
 }
 
-export function update_favicon(new_message_count, pm_count) {
+function clear_alternate_icon() {
+    $("#alternate_icon").attr("href", "");
```

Comment:
An empty string resolves as a relative URL to the current page. That’s not what you want.